Packs/AzureKeyVault/Integrations/AzureKeyVault/AzureKeyVault.yml

```diff
@@ -0,0 +1,1171 @@
+      name: certificates
+      predefined:
+      - get
+      - list
+      - update
+      - create
+      - import
+      - delete
+      - recover
+      - backup
+      - restore
+      - managecontacts
+      - manageissuers
+      - getissuers
+      - listissuers
+      - setissuers
+      - deleteissuers
+      - purge.
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Specifies whether Azure Virtual Machines are permitted to retrieve
+        certificates stored as secrets from the key vault. If the Key Vault exists,
+        you must supply the previous value inorder to keep it the same. Default value
+        is True.
+      isArray: false
+      name: enabled_for_deployment
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Specifies whether Azure Disk Encryption is permitted to retrieve
+        secrets from the vault and unwrap keys.If the Key Vault exists, you must supply
+        the previous value inorder to keep it the same. Default value is True.
+      isArray: false
+      name: enabled_for_disk_encryption
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Specifies whether Azure Resource Manager is permitted to retrieve
+        secrets from the key vault. If the Key Vault exists, you must supply the previous
+        value inorder to keep it the same. Default value is True.
+      isArray: false
+      name: enabled_for_template_deployment
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: The default action when no rule from ip_rules and from vnet_subnet_id
+        match . For example,  If no ip_rules and vnet_subnet_id arguments are defined,
+        the access to the key vault from any ip address or virtual network will be
+        accrodingly to the default_action value. If you wish to allow access only
+        from specific virtual network or ip address, use the ip_rules or the  vnet_subnet_id
+        arguments. This is only used after the bypass property has been evaluated.
+        Network acl property.
+      isArray: false
+      name: default_action
+      predefined:
+      - Allow
+      - Deny
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Tells what traffic can bypass network rules. This can be 'AzureServices'
+        or 'None'. For example, use 'AzureServices' if you wish to give azure services
+        access to key vault, although the default action is 'Deny' or the access for
+        a specific IP address. Network acl property. Default value is 'AzureServices'.
+      isArray: false
+      name: bypass
+      predefined:
+      - AzureServices
```

Comment:
only one option? can you please explain?

Reply:
I missed this one. bypass argument should accept also 'None' option.